gulpfile.js

```diff
@@ -161,6 +161,35 @@ gulp.task('minify', function() {
 });
 
 /**
+ * Minify the viewer's HTML, CSS and JS assets
+ */
+gulp.task('minifyViewer', function() {
+  var pipelines = {
+    'css': [cssmin({'keepSpecialComments': 0}), rev()],
+    'html': [minifyHtml({'empty': true})],
+    'js': [uglify(), rev()]
+  };
+
+  return gulp.src('./public/viewer/*.html')
```

Comment:
Is it required to include the full viewer or can we just pull it in through bower (e.g., https://libraries.io/bower/pdf.js-viewer) ?